src/api/app/models/user.rb

```diff
@@ -877,6 +874,10 @@ def watched_projects
     Project.where(id: watched_items.where(watchable_type: 'Project').pluck(:watchable_id)).order('LOWER(name), name')
   end
 
+  def regenerate_rss_secret
+    update!(rss_secret: SecureRandom.base58(24))
```

Comment:
We should explain why we didn't go with [has_secure_token](https://api.rubyonrails.org/classes/ActiveRecord/SecureToken/ClassMethods.html#method-i-has_secure_token). Otherwise, we definitely don't know in X months/years why this isn't the case.

Reply:
pushed a comment